use-cases/data-export.md

```diff
@@ -0,0 +1,84 @@
+
+## Initiate the export
+
+To initiate a bulk data export, use the [Add export](../operations/exports.md#add-export) operation, specifying the data _Entity type_, e.g. `OrderItem`.
+
+| <div style="width:350px">'How to' use case</div> | API Operations |
+| :-- | :-- |
+| How to create a new bulk data export | [Add export](../operations/exports.md#add-export) |
+
+## Check export status
+
+To monitor the progress of the export and retrieve the exported data, use the [Get all exports](../operations/exports.md#get-all-exports) operation. This operation provides a list of all ongoing and completed exports, including their status.
+
+| <div style="width:350px">'How to' use case</div> | API Operations |
+| :-- | :-- |
+| How to check on data export progress | [Get all exports](../operations/exports.md#get-all-exports) |
+
+## Download exported data
+
+Once the export status changes to `Success`, you can download the exported data from the provided [file URLs](../operations/exports.md#exported-file). For example:
+
+```javascript
+{
+    "Exports": [
+        {
+            "Id": "3fa85f64-5717-4562-b3fd-2c963f66afa6",
+            "Status": "Success",
+            "EntityType": "OrderItem",
+            "ExpiresUtc": "2023-10-26T11:42:28Z",
+            "Files": [
+                {
+                    "Url": "https://example.com/exports/3fa85f64-5717-4562-b3fd-2c963f66afa6-1.jsonl?example=signature",
+                    "SizeInBytes": 1215279
+                },
+                {
+                    "Url": "https://example.com/exports/3fa85f64-5717-4562-b3fd-2c963f66afa6-2.jsonl?example=signature",
+                    "SizeInBytes": 1398362
+                }
+            ]
+        },
+    ]
+}
+```
+> **Multiple files:** Note the data export may consist of multiple files.
+
+## Frequently Asked Questions
+
+### Will the supported entity types be extended in future?
+
+Depending on user demand, the **Mews Connector API** may be expanded to support additional data entities for bulk export.
+
+### How often should I check for updates?
+
+We do not provide specific guidance on this because the export duration can vary enormously. The current mechanism is considered a trial which may be enhanced in future. For example, we may move to a model whereby a webhook event is generated when the export is ready.
+
+### Can I export data from multiple enterprises in one request?
+
+Yes, you can use a [Portfolio Access Token](../guidelines/multi-property.md) to request data from multiple enterprises. Doing so will generate an export of data for all enterprises within scope of the Access Token.
+
+### What happens if I request a data export when there is already an export pending?
+
+The [Add export](../operations/exports.md#add-export) operation returns a 400 HTTP error code in that case.
```

Comment:
question: Should this limitation be mentioned in the Initiate the export section above?

Reply:
yes ok, agreed.